packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -83,19 +83,20 @@ protected function tearDown(): void
     }
 
     /**
-     * @deprecated Use \Rector\Testing\PHPUnit\AbstractRectorTestCase::yieldFilePathsFromDirectory() instead
+     * @return Iterator<<string>>
      */
     protected function yieldFilesFromDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
     {
         return FixtureFileFinder::yieldDirectory($directory, $suffix);
     }
 
     /**
-     * @return Iterator<<string>>
+     * @api
+     * @deprecated BC alias for yieldFilesFromDirectory()
      */
     protected function yieldFilePathsFromDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
```

Comment:
the `yieldFilePathsFromDirectory` never exists in 0.14.0 so should can be removed?

Reply:
I see. 

Then we'd need to upgrade all our packages to use the `yieldFilesFromDirectory()`. Could you handle it?

Reply:
yes, I will try to update the rector-* first.

Reply:
https://github.com/rectorphp/rector-src/pull/2894

Reply:
Awesome work, thank you :+1: